.github/workflows/cross-version-tests.yml

```diff
@@ -41,7 +41,7 @@ jobs:
   set-matrix:
     runs-on: ubuntu-latest
     timeout-minutes: 10
-    if: github.event_name != 'schedule' || github.repository == 'mlflow/mlflow'
+    if: github.event_name != 'schedule' || github.repository == 'mlflow/mlflow' || (github.event_name != 'pull_request' || github.event.pull_request.draft == false)
```

Comment:
Why do we need ()?

Reply:
Should it be (github.event_name == 'pull_request' && github.event.pull_request.draft == false)?

Reply:
I think () can be removed.